src/pykeen/nn/functional.py

```diff
@@ -14,13 +14,11 @@
 __all__ = [
     "circular_correlation",
     "quat_e_interaction",
+    "batched_dot",
```

Comment:
@mberr maybe just axe the functional module and put these all in `pykeen.nn.utils`?